### PR TITLE
Fix relations between models

### PR DESCRIPTION
**Pull request by AngelicaAgudelo**
Fix the DB and the relationship

### CODE REVIEW

File: back-end-yaz/src/server/src/models/index.js [hunk 1]
```diff
@@ -17,12 +18,12 @@ const sequelize = new Sequelize(
...
-                require: true,
-                rejectUnauthorized: false
-            },
-        }
+        // dialectOptions: {
+        //     ssl: {
+        //         require: true,
+        //         //rejectUnauthorized: false
+        //     },
+        // }
```
Comment (juanDuqueS) [inline back-end-yaz/src/server/src/models/index.js:26]: SSL commented

File: back-end-yaz/src/index.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ app.use('/services', serviceRoutes);
 app.use('/payment_services', payment_serviceRoutes);
 app.use('/payment_items', payment_itemRoutes);
 app.use('/items', itemRoutes);
+app.use('/service_request', service_requestRoutes);
```
Comment (juanDuqueS) [inline back-end-yaz/src/index.js:30]: the route must be plural

File: back-end-yaz/src/server/src/models/index.js (hunk 1 above)
Comment (AngelicaAgudelo) [reply back-end-yaz/src/server/src/models/index.js]: Fixed! Thanks!

File: back-end-yaz/src/index.js (hunk 2 above)
Comment (AngelicaAgudelo) [reply back-end-yaz/src/index.js]: Fixed! Thanks!

**Review by alejandroVaron — APPROVED** [review]
Nice

**Review by juanDuqueS — APPROVED** [review]

**Pull request merged**